tests/validation_suite/automation_framework/worker/worker_utility.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import logging
+import json
+
+from automation_framework.worker_lookup.worker_lookup_params \
+    import WorkerLookUp
+from automation_framework.worker_retrieve.worker_retrieve_params \
+    import WorkerRetrieve
+from automation_framework.worker_register.worker_register_params \
+    import WorkerRegister
+from automation_framework.worker_set_status.worker_set_status_params \
+    import WorkerSetStatus
+from automation_framework.worker_update.worker_update_params \
+    import WorkerUpdate
+from automation_framework.utilities.workflow \
+        import submit_request, validate_response_code
+
+logger = logging.getLogger(__name__)
+
+
+def submit_worker_actions(input_request, request_mode, tamper,
+                          output_json_file_name, uri_client,
+                          request_method, worker_obj,
+                          request_id):
+    ''' Function to submit worker actions.
+        Reads input json file of the test case.
+        Triggers create worker request, submit request and validate response.
+        Input Parameters : request, id_gen, output_json_file_name,
+        worker_obj, uri_client, check_worker_result
```

Comment:
Please provide description for input parameters.

Reply:
Done